code/controllers/subsystem/ticker.dm

```diff
@@ -135,6 +135,7 @@ SUBSYSTEM_DEF(ticker)
 			if(!mode.explosion_in_progress && mode.check_finished(force_ending) || force_ending)
 				current_state = GAME_STATE_FINISHED
 				toggle_ooc(1) // Turn it on
+				GLOB.dooc_allowed = TRUE // Previous line already announces OOC availability, also DOOC has no convenience procs anyways
```

Comment:
why not make one

Reply:
toggle_ooc is only called here and and when parsing the config, toggle_dooc would only be called here to do exactly what this one line does and nothing more.

Reply:
why not make one

Reply:
make it a proc Just In Case